src/main/java/org/opentripplanner/framework/text/MarkdownFormatter.java

```diff
@@ -79,7 +79,7 @@ public static String indentInTable(int level) {
   }
 
   public static String lineBreak() {
```

Comment:
```suggestion
  /**
   * Line-breaks in markdown is a bit problematic, we should avoid mixing HTML tags and markdown since not all 
   * Markdown applications support it, the same goes for ending the line with {@code \}. The best alternative
   * seams to be using two trailing spaces {@code "  "} followed by a line-break, even if this is error-prune.
   * This method does not add the line-break, just the trailing spaces. 
   */
  public static String lineBreak() {
```